src/data.rs

```diff
@@ -125,32 +156,6 @@ mod tests {
     }
 
     #[test]
-    fn data_payload_size() {
```

Comment:
Because `payload_size` was removed.
Instead, for size checks, the serialised data is now considered instead of just the contents.